devstack/local.conf.nfp

```diff
@@ -16,31 +16,35 @@ SERVICE_TOKEN=admin
 
 # NFP Service
 ENABLE_NFP=True
-# TODO(dpak): Change DEVSTACK_MODE to NFP_DEVSTACK_MODE
-# Edit DEVSTACK_MODE with either 'base' or 'advanced' or 'enterprise'
-[[ $ENABLE_NFP = True ]] && DEVSTACK_MODE=base
-[[ $DEVSTACK_MODE != base ]] && enable_plugin neutron-vpnaas https://git.openstack.org/openstack/neutron-vpnaas stable/mitaka
-[[ $DEVSTACK_MODE != base ]] && enable_plugin ceilometer https://git.openstack.org/openstack/ceilometer.git stable/mitaka
+# Edit NFP_DEVSTACK_MODE with either 'base' or 'advanced' or 'enterprise'
+[[ $ENABLE_NFP = True ]] && NFP_DEVSTACK_MODE=base
+[[ $NFP_DEVSTACK_MODE != base ]] && enable_plugin neutron-vpnaas https://git.openstack.org/openstack/neutron-vpnaas stable/mitaka
+[[ $NFP_DEVSTACK_MODE != base ]] && enable_plugin ceilometer https://git.openstack.org/openstack/ceilometer.git stable/mitaka
 
-if [[ $DEVSTACK_MODE != base ]]; then
+if [[ $NFP_DEVSTACK_MODE != base ]]; then
     GBPSERVICE_REPO=https://github.com/oneconvergence/group-based-policy.git
     GBPSERVICE_BRANCH=mitaka_21st_march_base
 else
     GBPSERVICE_REPO=https://git.openstack.org/openstack/group-based-policy
-    GBPSERVICE_BRANCH=refs/changes/45/309145/42
+    GBPSERVICE_BRANCH=master
 fi
 enable_plugin gbp $GBPSERVICE_REPO $GBPSERVICE_BRANCH
 
-if [[ $DEVSTACK_MODE != base ]]; then
+if [[ $NFP_DEVSTACK_MODE != base ]]; then
     # External Network
     EXT_NET_GATEWAY=
     EXT_NET_ALLOCATION_POOL_START=
     EXT_NET_ALLOCATION_POOL_END=
     EXT_NET_CIDR=
-    EXT_NET_MASK=
+
+    VyosQcow2Image=
+    Haproxy_LBaasV2_Qcow2Image=
```

Comment:
I understand Deepak addressed vyos image build. Is Haproxy_LBaasV2_Qcow2Image build is also done ?
We can address this in the new PR.